types/plotly.js/lib/traces/ohlc.d.ts

```diff
@@ -45,8 +45,8 @@ export type XCalendar =
     | 'thai'
     | 'ummalqura';
 
-export interface OhclData {
-    type: 'ohcl';
```

Comment:
This misspelled word on line 49 causes an error when I create an OHLC chart with Plotly.